src/main/java/com/gluonhq/NativeBaseMojo.java

```diff
@@ -137,6 +137,9 @@
     @Parameter(property = "gluonfx.remoteDir")
     String remoteDir;
 
+    @Parameter(property = "gluonfx.appIdentifier")
+    String appIdentifier;
```

Comment:
if this parameter is not provided, appIdentifier = null I suppose? (needed for the next change)

Reply:
Yes, correct